source/class/qx/ui/toolbar/CheckBox.js

```diff
@@ -24,27 +24,6 @@ qx.Class.define("qx.ui.toolbar.CheckBox",
 {
   extend : qx.ui.form.ToggleButton,
 
-
-
-
-  /*
-  *****************************************************************************
-     CONSTRUCTOR
-  *****************************************************************************
-  */
-
-  construct : function(label, icon)
-  {
-    this.base(arguments, label, icon);
-
-    // Toolbar buttons should not support the keyboard events
-    this.removeListener("keydown", this._onKeyDown);
-    this.removeListener("keyup", this._onKeyUp);
```

Comment:
This is true, but needed for more accessiblity support. I agree that some enhancements which are not required could be discussed in another pr, but imo this one is required and meant to be discussed in this pr

Reply:
Understood. Why don't you add a section to the qooxdoo.git/docs/release-notes.md that says that you added the accessibility support, and also documents the other enhancements included here. Since they're not backward compatible, they probably won't be back-ported into 6.x but rather be released with 7.0 where behavior changes are allowed (but should be documented in the release notes).

Reply:
Should i open another PR with the enhancements which are not required? In that PR i would document these then.
In this PR i will add an section to qooxdoo.git/docs/release-notes.md about (more) accessibility support to the mentioned components as suggested.
On a sidenote: Why are they not backward compatible?

Reply:
They changed the expected behavior of old applications. You can'nt be shure that anyone relies on this.
So you should split the PR into a part which aria support and a part with better accessiblity support if this is possible